config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json

```diff
@@ -29,7 +29,7 @@
+      "desc:8": "Sustainability. It\u0027s not the name of the game... but it could be!\n\nIf you\u0027ve ever wanted to generate many chemicals from a single source, you\u0027ve probably looked at Oil or Benzene. However there\u0027s also the Algae Farm. It\u0027s just a pond of sorts, so it doesn\u0027t need power to run. Since it also doesn\u0027t really follow normal recipes, you won\u0027t find any NEI recipe page for the Algae. The multi just produces them on a regular schedule.\n\nThe tier of the multi is given by the casings on the bottom layer of the multi, which all have to match to decide the tier. Each tier from ULV (starting at 50s per batch) till IV (which is 25s per batch) reduces the time it takes to produce Algae by about 5 seconds, and then beyond that each tier will cut the speed in half. The cases will currently work up to UHV cases, and casings beyond that will not work with this machine. Randomly the production speed can either double or triple the time it takes as well, but this happens relatively rarely.\n\nEach tier increase also adds to the amount of Algae you get in a single cycle, as well as the type of Algae produced. Each cycle gives out an exact amount as well as a bonus amount 10% of the time. The Algae Farm also accepts Compost, that you can make from the algae itself, to boost the multi\u0027s tier by one. This might be expensive at the start, but it\u0027s heavily recommended in most cases. You can get regular Algae and Green Algae starting at ULV, Brown Algae starts appearing at HV, Golden-Brown Algae at EV, and Red Algae at IV, though most of the cases where they start appearing they start doing so in very small quantities.\n\nTo find out more about the exact output quantities and timings, please see the wiki: §9§n[url]https://gtnh.miraheze.org/wiki/Algae_Farm[/url]§r\n\nThe Algae can be used for various chemicals like Methane, Ammonia, Sulfuric Acid, Sodium Carbonate, Ethylene, Lithium Chloride, and even some power-generating fluids. You typically need a Chemical Plant for that. In IV tier you can also combine Green Algae with Glowstone to get Glowing Coral and make Sunnarium from that. If you\u0027re curious about all the processing chains, look them up on NEI: quests to cover them are coming up, possibly with more recipes that don\u0027t need the chemplant."
```

Comment:
I thought the url tag alone is sufficient now without the §9§n...§r?

Reply:
I followed the convention found from #7 here https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/tree/master/config/betterquesting#some-more-guidelines-for-quests-development

I can change it if needed but it doesn't show up to the end user of course.

Reply:
steelux is fixing that guide as we speak (its a recent change in bq)

Reply:
you might also want to update BQ to see it work if you are not on the newest